.github/release-image.yaml

```
@@ -0,0 +1,55 @@
+name: release
+
+on: 
+  workflow_dispatch: {}
+  push:
+    tags:
+      - '*'
+
+permissions:
+  packages: write
+  contents: write
+  
+jobs:
+  run-goreleaser:
+    runs-on: ubuntu-24.04
+    needs:
+      - release
+      
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: all
```

I think the default for the platforms variable is 'all.' :+1: 
→ https://github.com/docker/setup-qemu-action?tab=readme-ov-file#customizing

```suggestion
        uses: docker/setup-qemu-action@v3
```